cmd/mosn/main/control.go

```diff
@@ -168,8 +168,18 @@ var (
 	cmdStop = cli.Command{
 		Name:  "stop",
 		Usage: "stop mosn proxy",
-		Action: func(c *cli.Context) error {
-			return nil
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:   "config, c",
+				Usage:  "load configuration from `FILE`",
+				EnvVar: "MOSN_CONFIG",
+				Value:  "configs/mosn_config.json",
+			},
+		},
+		Action: func(c *cli.Context) (err error) {
+			mosnConfig := configmanager.Load(c.String("config"))
+			mosn.InitDefaultPath(mosnConfig)
+			return stagemanager.StopMosnProcess(mosnConfig)
```

Comment:
```suggestion
			app := mosn.NewMosn()
			stm := stagemanager.InitStageManager(c, c.String("config"), app)
			return stm.Stop()
```
Maybe this style could be better, align up the existing usage.

Reply:
stm.Stop() method  has existed, so I keep StopMosnProcess() method name.

Reply:
I'm fine with `StopMosnProcess`, I just think `Stop` could be better personally.

The more important thing is: we'd better use stm as a module, not just a util function.